Resources/Prototypes/Entities/Objects/Tools/welders.yml

```
@@ -168,16 +168,16 @@
       Welder:
         reagents:
         - ReagentId: WeldingFuel
-          Quantity: 250
-        maxVol: 250
+          Quantity: 500
+        maxVol: 500
   - type: Tool
-    speedModifier: 1.3
+    speedModifier: 1.5
 
 - type: entity
   name: experimental welding tool
   parent: Welder
   id: WelderExperimental
-  description: "An experimental welder capable of self-fuel generation and less harmful to the eyes."
+  description: "An experimental welder capable of self-fuel generation, an extremely hot flame, and is less harmful to eyes."
```

The structure of this sentence isn't quite right. I'd change it to something along the lines of:
```suggestion
  Description: "An experimental welder that is less harmful to the eyes while having a hotter flame and slowly regenerating fuel."
```

---

To be fair to you, the original sentence contained the same issue.